new/LIT6186JamSarAaron.xml

```diff
@@ -0,0 +1,64 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6186JamSarAaron" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ድርሳን፡ ዘደረሰ፡ ብፁዕ፡ ወቅዱስ፡ አባ፡ ያዕቆብ፡ ዘሴሩግ፡ በእንተ፡ ዕረፍቱ፡ ለአሮን፡ ካህን፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Dǝrsān za-darasa bǝḍuʿ wa-qǝddus ʾabbā
+                Yaʿǝqob ze-Serug baʾǝnta ʿǝraftu la-ʾAron kāhǝn</title>
```

Comment:
za

Reply:
It also might be good to normalize the spelling to Śǝrug for the sake of searches.

Reply:
Ahh, sorry, I mean "ያዕቆብ፡ ዘሥሩግ፡ / Yāʿǝqob za-Śǝrug / Jacob of Sarug", I think this is what most of the references on BM have used.

Reply:
The ms. has ሴሩግ Serug without any doubt. I suggest we keep Sarug in the translation as the most common form under which a user is most likely to search.

Reply:
I think that the reading of the manuscript should be kept in the geez title as well as in the transliteration so far, the work entry has the english title and an author attribution which makes it findable anyway